src/main/com/example012/v1/LotteryGame.java

```diff
@@ -0,0 +1,79 @@
+package main.com.example012.v1;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+import static main.com.example012.v1.StringConstant.*;
+
+public class LotteryGame {
+
+    private Company gameType;
+
+    private Tickets winningTicket;
```

Comment:
Nazev classy Tickets představuje množné číslo ... winningTicket je jednotné, něco nesedí. Classy by měly v drtivé většině představovat jednodnté číslo ... pokud pak chceš víc tiketů, tak se to nacpe do nějaké kolekce